stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/CSS.java

```diff
@@ -27,12 +28,15 @@
 
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 public class CSS implements ChromeDevtoolsDomain {
   private final ChromePeerManager mPeerManager;
   private final Document mDocument;
   private final ObjectMapper mObjectMapper;
+  private final Map<String, ArrayList<CSSProperty>> mSelectorCollector = new HashMap<>();
```

Comment:
please put a javadoc on this explaining how this is used and the restrictions on it -- certainly don't want a future maintainer using it on another thread, or not realizing that there is a "protocol" attached with correctly using it!